index.js

```diff
@@ -8,6 +8,10 @@ const MergeTrees = require('broccoli-merge-trees');
 module.exports = {
   name: '@smile-io/ember-polaris',
 
+  isDevelopingAddon() {
+    return process.env.SMILE_DEV;
```

Comment:
Do we want something more general than `SMILE_DEV` on this? Or are we only allowing Smile team to develop this addon? 😅

Reply:
I think the idea is to be something more unique to avoid anyone having this by mistake and have this result in files being watched and increase build times

Reply:
of course, anyone can set this flag if their developing 😄

Reply:
Yeah I know 😛